packages/astro-rss/src/index.ts

```diff
@@ -47,6 +47,16 @@ type RSSFeedItem = {
 	customData?: z.infer<typeof rssSchema>['customData'];
 	/** Whether draft or not */
 	draft?: z.infer<typeof rssSchema>['draft'];
+	/** Categories or tags related to the item */
+	categories?: z.infer<typeof rssSchema>['categories'];
+	/** The item author's email address */
+	author?: z.infer<typeof rssSchema>['author'];
+	/** A URL of a page for comments related to the item */
+	comments?: z.infer<typeof rssSchema>['comments'];
```

Comment:
nit: `commentsUrl`?

Reply:
That's a good idea. I was trying to stay close to the RSS field names. Though I did change `category` to `categories` to deal with the plural. This would make it more obvious and I'll update that.